spec/models/donation_spec.rb

```diff
@@ -1,5 +1,54 @@
 require 'rails_helper'
 
 RSpec.describe Donation, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context "is valid with proper attributes" do
+    let(:user) {
+      User.create(first_name: "Donald", last_name: "Trump", email: "trump@luxury.com", username: "therealtrump", password: "password", password_confirmation: "password")
+    }
+    let(:candidate_issue) {
+      CandidateIssue.create
+    }
+
+    let(:valid_attributes) do
+      {
+                    amount: 100,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.